diffblue-base.yml

```diff
@@ -1,14 +0,0 @@
-# This file contains configuration for Diffblue Cover that is specific for
-# this project, typically a custom buildCmd, classpathCmd and
-# java-version (as a CBMC argument).
-#
-# Other general options such as skipVerification, ignoreExistingCoverage, CBMC
-# arguments (except for java-version) and phases SHOULD NOT be specified here.
-# They are part of diffblue.yml file that is automatically generated by
-# merging a custom yml file on top of this one using the script
-# in dashboard/scripts/update-diffblue-yml. Any options specified here would
-# be overriden to keep consistency with other benchmarks.
-buildCmd: cd librec; mvn -e -X -Dmaven.test.skip=true compile
```

Comment:
Similarly question as here: https://github.com/Diffblue-benchmarks/Dragonfly/pull/10#discussion_r328667796

Reply:
Answered in the linked comment